src/core/file.js

```diff
@@ -36,6 +37,19 @@ module.exports = {
     });
   },
 
+  move(filePath, targetDir, extraFile, logger = defaultLogger) {
+    logger.info(`Moving the ${filePath} into ${targetDir}.`);
+    const target = path.join(targetDir, extraFile.path);
+    // Only move file with override status
+    if (extraFile.config === 'override') {
+      fs.move(filePath, target, (err) => {
+        if (err) return logger.info(`Can not move Test Suite ${err}`);
+        logger.info('Updated script repository with Test Suite from TestOps');
```

Comment:
Can you log the message containing extraFile's content?

Reply:
The new message will be "Updated script repository with extra files from TestOps" ?

Reply:
Moved the ${filePath} into ${targetPath}
Can not move ${filePath} into ${targetPath}

Reply:
Ok a